common/lib/xmodule/xmodule/services.py

```diff
@@ -119,3 +121,78 @@ def get_teams_configuration(self, course_id):
         if not self._course:
             self._course = self.get_course(course_id)
         return self._course.teams_configuration
+
+
+class CallToActionService:
```

Comment:
Cale, I dunno if this is the best place for this class. Let me know where you want it.

Reply:
So, I was thinking that this should probably be pluggable, so that individual features can add responses for specific CTA positions. I'll probably move the base definition into `openedx` somewhere, and the specific PLS plugin into `lms` somewhere.